stubs/docopt/docopt.pyi

```diff
@@ -0,0 +1,11 @@
+from typing import Any, Optional, Union
+
+__version__: str
+
+def docopt(
+    doc: str,
+    argv: Optional[list[str]] = ...,
+    help: Optional[bool] = ...,
```

Comment:
`Optional[bool]` means "`True`, `False` or `None`". I don't see how `None` would be useful: `docopt.py` defines this as `def docopt(..., help=True, ...)`, and the docstring says `bool` too.

```suggestion
    help: bool = ...,
```

Reply:
Oops, I was mistakenly interpreting `Optional` to mean "This is an optional argument". I see now, thanks, fixed.